frontend/src/lib/dashboard/getQueryResults.ts

```diff
@@ -33,9 +33,11 @@ export async function GetMetricQueryRange(
 	);
 
 	if (response.statusCode >= 400) {
-		throw new Error(
-			`API responded with ${response.statusCode} -  ${response.error} status: ${response.message}, errors: ${response?.body}`,
-		);
+		let error = `API responded with ${response.statusCode} -  ${response.error} status: ${response.message}`;
+		if (response.body) {
```

Comment:
is the response body is `{}` this would still add `errors:{}` in the display message

Reply:
Done.